src/App.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
 
+const preparedTodos = [...todos.map((todo) => {
```

Comment:
```suggestion
const preparedTodos = todos.map((todo) => {
```
map повертає новий масив, spread оператор зайвий